ChangeLog

```diff
@@ -1,3 +1,7 @@
+== 2025-04-11 version 0.3.29
+
+* Support Ruby 2.7 to 3.4
```

Comment:
[suggestion] Just to confirm, will Rubies less than 2.7 be dropped for the support? If yes, describing that might be a bit more helpful for users, e.g.

```suggestion
* Requires Ruby 2.7 or higher. Support for versions earlier than 2.7 has been removed.
```

In addition, we might need to consider adding `required_ruby_version` to the gemspec, e.g.

```ruby
spec.required_ruby_version = '>= 2.7.0'
```


Ref https://github.com/treasure-data/td-logger-ruby/compare/v0.3.28...14ea0497e0e2157964e83d7faf0682eac65484aa

Reply:
Make sense! Although I suspect ruby < 2.7 can still work, it's best if we make it explicit with `required_ruby_version`. Thank you for pointing it out!